doc/source/user/use-custom-patches.rst

```diff
@@ -0,0 +1,23 @@
+===================
+Use custom patches
+===================
+
+
+Sometimes there may be upstream patches that you want to apply with the dev-patcher
+but creating a PR and send it to the soco8ks repo may delay your work, or you are
```

Comment:
That's not the reason. Please remove this line.

You could rephrase to simply:

```If you want to apply upstream patches in your environment, set the following: ...```

```This way, the patches will only be carried in your environment. If you want to change the product instead (for developer mode or not), please propose a PR to the socok8s repo.```